infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/KZGException.java

```diff
@@ -11,15 +11,11 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.kzg.impl;
+package tech.pegasys.teku.kzg;
 
-public class KzgException extends IllegalArgumentException {
+public class KZGException extends RuntimeException {
 
-  public KzgException(String message) {
```

Comment:
I'd leave this one, we have some logic in wrapper we may want to react without having throwable